### DISCUSSION TITLE
Alterar comportamento searchService do componente po-dynamic-form

### DISCUSSION THREAD

**Discussion by wreliquiasTR** [po-ui/po-angular#760] [Ideias]
Preciso usar o tag searService do componente po-dynamic-form. Hoje ele permite buscar dados de uma url, gostaria de alterar para que seja possivel usar uma função que retorne esses dados; É possivel eu ter uma orientação de onde e como eu mudaria para o componente passar a trabalhar da maneira que eu preciso?

**Comment by jhonyeduardo** [comment]
Boa tarde @wreliquiasTR 

Atualmente não é possível, o ideial seria abrir uma issue de melhoria para avaliarmos esta sugestão.
